.github/workflows/ci.yml

```diff
@@ -59,12 +59,13 @@ jobs:
         if: success()
         run: Scripts/upload-coverage-reports.sh ${{ matrix.platforms }}
   spm-12:
-    name: SPM Build Xcode 12
-    runs-on: macOS-10.15
+    name: Build Xcode 12
+    runs-on: macOS-11
     strategy:
       matrix:
         platforms: [
           'iOS_14,tvOS_14,watchOS_7',
+          'macOS_11',
```

Comment:
Now that we're running on macOS 11 and targeting Xcode 12 we can run tests against the macOS 11 SDK!